demisto_sdk/commands/validate/file_validator.py

```diff
@@ -130,6 +130,9 @@ def get_modified_files(files_string, tag='master', print_ignored_files=False):
                 file_status = 'r'
                 file_path = file_data[2]
 
+            if not os.path.isfile(file_path):
```

Comment:
how is this tested? what is the reason that a file will not have a path? I am not sure about this.